subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -39,14 +39,193 @@
         <xsl:apply-templates select="/book" mode="object.title.markup.textonly"/>
     </xsl:template>
 
+    <xsl:template name="chunk-element-content">
+        <xsl:param name="content">
+            <xsl:apply-imports/>
+        </xsl:param>
+
+        <html>
+            <xsl:call-template name="html.head"></xsl:call-template>
+            <body>
+                <xsl:call-template name="header.navigation"></xsl:call-template>
+                <nav class="docs-navigation">
+                    <ul>
+                        <li><a href="/current/userguide/userguide.html">Home</a></li>
```

Comment:
❓ won't `/current` in those links cause navigation issues when published as nightly/rc or when the documentation is archived for a past release?

Reply:
@eskatos We do URL rewriting with JavaScript for all navigation through dotcom-docs because docs is not aware of other docs versions. This works the same as it does today for the top navigation. 

However, your suggestion has helped me decide that this JS should live in gradle/gradle and not in another project.

Reply:
You know what, I was wrong here. We have different code for the header we inject into old docs versions that has `/current` URLs; this is not what we do for Gradle 4.4+ where the header isn't injected. You were right to call this out.